secant/Features/BackupFlow/Views/RecoveryPhraseBackupValidationView.swift

```diff
@@ -0,0 +1,272 @@
+//
+//  RecoveryPhraseBackupView.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 10/29/21.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct RecoveryPhraseBackupValidationView: View {
+    let store: RecoveryPhraseValidationStore
+
+    var body: some View {
+        WithViewStore(self.store) { viewStore in
+            VStack {
+                header(for: viewStore)
+                    .padding(.horizontal)
+                    .padding(.bottom, 10)
+                ZStack {
```

Comment:
Can you add spaces between all these views, it makes it easier to see what's going on for the overall view structure.